packages/blade/package.json

```diff
@@ -138,6 +138,7 @@
     "use-presence": "1.1.0",
     "@use-gesture/react": "10.2.24",
     "@floating-ui/react": "0.25.4",
+    "react-hot-toast": "2.4.1",
```

Comment:
Would RN consumers need to install this too even though they won't be using it at all?

Reply:
Dependencies will anyways be installed regardless, since most of your deps are inside "dependencies"